### DISCUSSION TITLE
MarbleNet vs noise?

### DISCUSSION THREAD

**Discussion by seekingdeep** [NVIDIA-NeMo/NeMo#3398] [General]
Hi there,
@fayejf @titu1994 @borisgin

I am interested in knowing the performance of Nemo's VAD "MarbleNet" on [the following sample.](https://www.dropbox.com/s/lkngxlybrh4zamh/example_vad_music.wav)
Basically, this is to detect the robustness of the VAD model against noise and other sounds that are active at the same time.
waiting for your reply

**Comment by titu1994** [comment]
You should be able to evaluate the sample yourself ? The models are public.

**Comment by seekingdeep** [comment]
In that case, you can close the discussion
